docs/user/SingleStore.md

```diff
@@ -8,9 +8,27 @@ If your SAS software order included SAS Viya with SingleStore, additional requir
 
 ## Deploying SAS Viya with SingleStore Using SAS Viya 4 Deployment
 
-You can deploy SAS Viya with SingleStore into a Kubernetes cluster that is running in Microsoft Azure. The [SAS Viya 4 Infrastructure as Code (IaC) for Microsoft Azure](https://github.com/sassoftware/viya4-iac-azure) GitHub project can automatically provision the required infrastructure components that support SAS Viya with SingleStore deployments. 
+You can deploy SAS Viya with SingleStore into a Kubernetes cluster running under:
+- Azure Kubernetes Service (AKS) in Microsoft Azure
+- Elastic Kubernetes Service (EKS) in Amazon Web Services
```

Comment:
Elastic Kubernetes Service (EKS) in Amazon Web Services (AWS)

Reply:
added (AWS)